relay-cabi/src/processing.rs

```diff
@@ -124,8 +124,7 @@ pub unsafe extern "C" fn relay_store_normalizer_normalize_event(
         measurements_config: None, // only supported in relay
         breakdowns_config: None,   // only supported in relay
         normalize_user_agent: config.normalize_user_agent,
-        normalize_transaction_name: false, // only supported in relay
-        tx_name_rules: &[],                // only supported in relay
+        transaction_name_config: Default::default(), // only supported in relay
```

Comment:
Instead of adding a third transaction name related parameter to `LightNormalizationConfig`, I decided to put it in its own struct.